src/Controller/UI/NationalDataCenterController.php

```diff
@@ -49,8 +49,7 @@ public function defaultAction($id = null)
 
         return $this->render('template/NationalDataCenter.html.twig', array(
             'NationalDataCenter' => $nationalDataCenter,
-            'form' => $form->createView(),
-            'entityService' => $this->entityHandler
```

Comment:
Line 40 above also asks for entityHandler

Reply:
Fixed it